### ISSUE TITLE
Progress is not updated

### ISSUE THREAD

**Issue by pvroosendaal** [mockingbot/react-native-zip-archive#184]
Hi,

Wonderful utility, working great! However, when I'm zipping a folder with images and video's, the progress is not updated in the subscribe event.
I'm getting progress 0 at start and 1 when it's completed, but nothing in between.

Can you help out?

I'm on version 5.0.0, testing on iPhone X with iOS 13.3

**Comment by plrthink** [comment]
could you show the snippet you're using?

**Comment by pvroosendaal** [comment]
I have a FileService class with the following function, where `subscribe(callback)` is the subscribe function from this lib.

```
subscribeToArchiving(callback: ({ progress: number, filePath: string })) {
    this.archiveProgressEvent = subscribe(callback)
}
```

And I'm calling that with the following snippet:

```
this.fileService.subscribeToArchiving(({ progress, filePath }) => {      
    console.log("subscribeToArchiving()", progress)
    this.setState({ archiveProgress: progress })
})
```

**Comment by plrthink** [comment]
Oh, sorry for the late response. Actually we haven't implemented the real progress event for archiving on iOS yet. It's just the faked version.

But it seems to be easy to accomplish. I would like to introduce this in the next release.

**Comment by pvroosendaal** [comment]
Ok, thanks for the info. Hope to see this enhancement soon!

**Comment by plrthink** [comment]
Hi, I just published version 5.0.2-beta.1 which includes this feature. Can you try it out? Please note that currently, I cannot emit the file being zipped out, so the `filePath` passed to the callback would always the empty string.

**Comment by plrthink** [comment]
closed via version 5.0.2
